client/src/css/Home.scss

```diff
@@ -1 +1,82 @@
 @import '../styles/colors.scss';
+
+.sessionCard {
+  box-shadow: none !important;
+  text-align: left !important;
+  width: 400px !important;
+  height: 100px !important;
+  margin-bottom: 20px !important;
+  border-radius: 15px !important;
+  transition: all 0.3s ease !important;
+  // box-shadow: 3px 3px 6px grey !important;
+}
+
+.ui.card > .content > .header {
+  font-family: Tommy !important;
+}
+.sessionCard:hover {
+  transition: all 0.3s ease;
+  background-color: #b2caee !important;
+  box-shadow: 3px 3px 6px grey !important;
+}
+
+.ui.button {
+  font-family: Tommy;
+  padding: 5px 27px;
+  margin-left: 280px;
+  transition: all 0.3s ease;
+}
+
+.homeHeader {
+  margin-top: 50px;
+  margin-left: 100px;
+  margin-bottom: 50px;
+  font-size: 80px;
+  text-align: left;
+  font-family: Tommy;
+}
+h1 {
+  font-family: Tommy;
+}
+.insideCard {
+  margin-top: 5px !important;
+  padding: 20px !important;
+  font-family: Tommy !important;
+}
+#join-btn {
+  padding: 5px 27px;
+  margin-left: 280px;
+  transition: all 0.3s ease !important;
+  font-family: Tommy !important;
+}
+#join-btn:hover {
+  transition: all 0.3s ease !important;
+}
+
+.ui.header {
+  font-family: Tommy;
+}
+
+.ui.header:not(h1):not(h2):not(h3):not(h4):not(h5):not(h6) {
+  font-size: 30px;
+  padding-top: 10px;
+  padding-left: 10px;
+}
+#form-text {
+  padding-left: 10px;
+}
+.ui.modal {
+  background-color: #c5cad4 !important;
+  font-family: Tommy;
+  width: 500px;
+}
+.ui.modal > .content {
+  background: #c5cad4;
+  padding-left: 25px;
+  padding-bottom: 30px;
+}
+
+.ui.modal > :last-child {
+  border-bottom-left-radius: 100px;
+  border-bottom-right-radius: 100px;
+}
```

Comment:
Nest these rules as well, and extract them (and other relevant css rules) into a separate DetailsModal.scss